packages/attention/stories/Attention.stories.tsx

```diff
@@ -163,8 +162,8 @@ export function Tooltip() {
 
 export function Popover() {
```

Comment:
Should we add `flip` to this story's Attention, so the component follows the button when screen is resized? Otherwise `autoUpdatePosition` won't trigger because we check if `isShowing && flip && targetEl && attentionEl` in the respective useEffect. Or maybe `flip` missing shouldn't prevent the auto update anyway?

Here's how this story behaves now:
![popover](https://github.com/warp-ds/react/assets/41303231/e5d8a62d-ce34-4166-ae3c-fdf6363568f9)

Reply:
Updated in this commit: [d22f435](https://github.com/warp-ds/react/pull/215/commits/d22f435b3891882e6751998ae83945e343665946)

Reply:
Just wondering, why do we need to check if `flip` is truthy when running autoUpdate? Would it break something if we run it with `flip` equal `false` or `undefined`?

Reply:
I noticed that if you run `autoUpdate` and don't specify how the `attentionEl` should reposition to stay in view as long as possible, then if you, for example set the `placement` to `right` or `left` and then start scrolling down, the `attentionEl` will continue to stay in view, even when the `targetEl` is no longer in view. At first, I thought that was a bug, but I then found out that the `[flip()](https://floating-ui.com/docs/flip)` function in floating-ui gives you the option to specify what should happen if the attentionEl doesn't fit anymore on the screen. So if you keep calling `autoUpdate` it will keep on recalculating the `attentionEl`'s position to keep it in view, but it doesn't know which `placement` we want it to fallback to if the original `placement` doesn't fit anymore, so it stays with the original `placement`. This isn't an issue if the `attentionEl` has `placement` `top` or `bottom`, only on the sides.

Reply:
Although, I just found now that you can also use floating-ui's [autoPlacement()](https://floating-ui.com/docs/autoPlacement), which we could use when flip is not enabled. `flip()` gives the user more options to configure how the attentionEl should reposition itself when there is no space, but if we want to call `autoUpdate` even when flip is not enabled, then we could call `autoPlacement`.

Reply:
I think that would be nice because otherwise users might not know they need to specify `flip` to get any auto updating. They would need to check the Attention logic to see that `flip` is needed in their setup for that. But I don't know how much work that requires, so maybe setting `flip` to `true` by default IF other conditions are met would be an alternative? It's hard to remember all conditions here so ignore me if my suggestion doesn't make any sense :D

Reply:
Yes I agree! I'll add `autoPlacement` and make sure that it will be enabled if the user hasn't enabled `flip`. I tried to set default value to true for `flip` but then I found issues with web components not working if you want to set `flip`to false in the markup, see this [issue](https://stackoverflow.com/questions/60678891/how-can-i-change-a-boolean-property-in-lit-element-to-hide-content-on-my-compone). So that's why I had to set `flip` as default `false`.

Reply:
Ah, great!